src/sagemaker/jumpstart/curated_hub/utils.py

```diff
@@ -0,0 +1,203 @@
+        str: The name of the default bucket. If the name was not explicitly specified through
+            the Session or sagemaker_config, the bucket will take the form:
+            ``sagemaker-hubs-{region}-{AWS account ID}``.
+    """
+
+    region: str = sagemaker_session.boto_region_name
+    account_id: str = sagemaker_session.account_id()
+
+    # TODO: Validate and fast fail
+
+    return f"sagemaker-hubs-{region}-{account_id}"
+
+
+def create_hub(
+    hub_name: str,
+    hub_description: str,
+    hub_display_name: str = None,
+    hub_search_keywords: Optional[List[str]] = None,
+    hub_bucket_name: Optional[str] = None,
+    tags: Optional[List[Dict[str, Any]]] = None,
+    sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
+) -> str:
+    """Creates a SageMaker Hub
+
+    Returns:
+        (str): Arn of the created hub.
+    """
+
+    if hub_bucket_name is None:
+        hub_bucket_name = _generate_default_hub_bucket_name(sagemaker_session)
+    s3_storage_config = {"S3OutputPath": hub_bucket_name}
+    response = sagemaker_session.create_hub(
+        hub_name, hub_description, hub_display_name, hub_search_keywords, s3_storage_config, tags
+    )
+
+    # TODO: Custom error message
+
+    hub_arn = response["HubArn"]
+    return hub_arn
+
+
+def describe_hub(
+    hub_name: str, sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION
+) -> Dict[str, Any]:
+    """Returns descriptive information about the Hub"""
+    # TODO: hub_name validation and fast-fail
+
+    response = sagemaker_session.describe_hub(hub_name=hub_name)
+
+    # TODO: Make HubInfo and parse response?
+    # TODO: Custom error message
+
+    return response
+
+
+def delete_hub(hub_name, sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION) -> None:
+    """Deletes a SageMaker Hub"""
+    response = sagemaker_session.delete_hub(hub_name=hub_name)
+
+    # TODO: Custom error message
+
+    return response
+
+
+def import_hub_content(
+    document_schema_version: str,
+    hub_name: str,
+    hub_content_name: str,
+    hub_content_type: str,
+    hub_content_document: str,
+    hub_content_display_name: str = None,
+    hub_content_description: str = None,
+    hub_content_version: str = None,
+    hub_content_markdown: str = None,
+    hub_content_search_keywords: List[str] = None,
+    tags: List[Dict[str, Any]] = None,
+    sagemaker_session: Session = DEFAULT_JUMPSTART_SAGEMAKER_SESSION,
+) -> Dict[str, str]:
+    """Imports a new HubContent into a SageMaker Hub
+
+    Returns arns for the Hub and the HubContent where import was successful.
+    """
+
+    response = sagemaker_session.import_hub_content(
+        document_schema_version,
+        hub_name,
+        hub_content_name,
+        hub_content_type,
+        hub_content_document,
+        hub_content_display_name,
+        hub_content_description,
+        hub_content_version,
+        hub_content_markdown,
+        hub_content_search_keywords,
+        tags,
```

Comment:
for this many arguments, can we use `kwarg` style args? `hub_name=hub_name, ...`

Reply:
Good point. Will do.